File: lessons/compiled_lang/java/src/g20.array.java
// Not public - see a00.output.java for why.
class G20Array {
    public static void main(String[] args) {
        String[] nicknames = {"bob", "ed", "steve", "ralph", "joe", "deb", "kate"};

        System.out.println("The names are: ");
        for (int i = 0; i < nicknames.length; i++) {
            System.out.println(" nicknames[" + i + "]=" + nicknames[i]);
        }
    }
}

File: lessons/compiled_lang/go/src/g20.array.go
package main

import "fmt"

func main() {
	nicknames := []string{"bob", "ed", "steve", "ralph", "joe", "deb", "kate"}

	fmt.Println("The names are: ")
	for i, name := range nicknames {
		fmt.Printf(" nicknames[%d]=%s\n", i, name)
	}
}
